app/serializers/order_serializer.rb

```diff
@@ -62,13 +68,44 @@ def ordered_by_current_user
     object.user == current_user
   end
 
+  def current_user_debt_permitted
+    current_user.total_debt.to_f > Dish::MAX_DEBT
+  end
+
   def from_today
     object.from_today?
   end
 
+  def order_owner
+    object.user.name
+  end
+
+  def current_user_ordered_dishes
+    return "You didn't order any dish." if current_user_dishes.blank?
```

Comment:
nit:
WDYTA `current_user_dishes.map(&:name).join(', ') || 'You didn't order any dish.'` here?